components/ImageButton/ImageButton.js

```diff
@@ -0,0 +1,37 @@
+
+import React, { PureComponent } from 'react';
+import { TouchableOpacity, StyleSheet, ImageBackground } from 'react-native';
+
+type Props = {
+    children: Object,
+    onPress: () => any,
+    style?: ComponentStyles
+};
+
+export default class ImageButton extends PureComponent<Props> {
+    render() {
+        const { onPress } = this.props;
+        const image = require('../../assets/threeDots.png');
```

Comment:
You can use import, but will be better to use vector icons, we already use https://github.com/oblador/react-native-vector-icons